### PR TITLE
Update README as in General Grants repo

### PR DESCRIPTION
**Pull request by alxs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,25 +29,51 @@
...
+- Lastly, we do not fund projects that actively encourage gambling, illicit trade, money laundering or criminal activities in general.
 
-The Web3 Foundation Open Grants Program focuses on technology grants up to **USD 30k for initial grants** and **USD 100k for follow-up grants**, which are tracked **transparently** on GitHub and disbursed in **bitcoin or DAI**. For larger grants, payments in other currencies and partly private submissions, please visit our [Web3 Foundation General Grants page](https://github.com/w3f/General-Grants-Program) on GitHub.
+Additionally to the information provided on your application, note that your project will need to comply with our [Guidelines for Milestone Deliverables](grants/milestone-deliverables-guidelines.md). In particular, we require all projects to create documentation that explains how their project works. At a minimum, _written_ documentation is required for funding. Tutorials or videos are also helpful for new users to understand how to use your product. If your team's background is purely technical, consider hiring a freelance or part-time technical writer as part of the grant.
```
Comment (alxs) [inline README.md:58]:
```suggestion
In addition to the information provided on your application, note that your project will need to comply with our [Guidelines for Milestone Deliverables](grants/milestone-deliverables-guidelines.md). In particular, we require all projects to create documentation that explains how their project works. At a minimum, _written_ documentation is required for funding. Tutorials or videos are also helpful for new users to understand how to use your product. If your team's background is purely technical, consider hiring a freelance or part-time technical writer as part of the grant.
```

**Comment by semuelle** [comment]
Hi @alxs, thanks for your delivery. Could you please sign the CLA? ;)

**Review by semuelle — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -72,40 +100,30 @@ Evaluators are individuals able to evaluate the technology delivered as a result
...
-4. **Milestone Review:**
-   1. [Evaluators](#w3f-grants-evaluators) can (and usually do) issue comments and request changes on the pull request.
-   2. Grants Evaluators will merge your pull request to accept the deliverable.
-5. **Milestone Payment:**
-   1. The [Operations Team](#w3f-operations-team) receives a notification once the delivery was accepted. As soon as feedback is provided by the evaluators, this feedback first needs to be resolved.
-   2. Operations Team makes the payment to the Bitcoin or Ethereum address specified in the initial application. The exchange rate for BTC depends on the time of payment.
-
-_The grant application can be amended at any time. However, this necessitates a reevaluation by the Committee._
-
-Additionally, we try to enforce the [code of conduct](https://github.com/w3f/Open-Grants-Program/blob/master/CODE_OF_CONDUCT.md) and based on this may [block users](https://github.blog/2016-04-04-organizations-can-now-block-abusive-users/).
```
Comment (semuelle) [inline README.md:108]: Is this information actually in the delivery readme?

Comment (semuelle) [reply README.md:108]: just checked. It is, but the links are dead because they are relative.

Comment (alxs) [reply README.md:108]: Thanks for noticing, will fix it.

**Comment by Noc2** [comment]
@alxs Could you double check the links?

**Comment by alxs** [comment]
I fixed them in w3f/Grant-Milestone-Delivery#109, just double-checked and everything works.

**Pull request merged**